src/auro-menu.js

```diff
@@ -117,15 +136,31 @@ class AuroMenu extends LitElement {
   makeSelection() {
     const option = this.items[this.index];
 
-    // only handle options that are not disabled
-    if (!option.disabled && !option.hidden) {
-      this.resetOptionsStates();
-      this.handleLocalSelectState(option);
-      this.dispatchEvent(new CustomEvent('selectedOption', {
-        bubbles: true,
-        cancelable: false,
-        composed: true,
-      }));
+    // only handle options that are not disabled, hidden or static
+    if (option && this.optionInteractive(option)) {
+      // fire custom event if defined otherwise make selection
+      if (option.hasAttribute('event')) {
+        this.dispatchEvent(new CustomEvent(option.getAttribute('event'), {
+          bubbles: true,
+          cancelable: false,
+          composed: true,
+        }));
+
+        this.dispatchEvent(new CustomEvent('customEventFired', {
+          bubbles: true,
+          cancelable: false,
+          composed: true,
+        }));
```

Comment:
What is the value of this? Should this be renamed to `auroMenuCustomEventFired`? I feel like the generic name can be confusing to a consumer? 

Like, maybe we will have another element that has a custom event being fired... know what I mean?

Reply:
I can make it be `auroMenuCustomEventFired`. I needed to also fire this one with the constant name in addition to whatever the custom event was that was defined. That way combobox (and select) can listen for this event and know that it should close the bib just like it does when `selectedOption` is fired.

@blackfalcon I changed the name but when I rebuild it no longer populates the event type in the API.md. Not sure why that would be just for changing the name. Any ideas?

Reply:
`auroMnuCustomEventFired` typo

https://github.com/AlaskaAirlines/auro-menu/pull/121/files#diff-764d55f08ee0afe3ee3fe4d5cd7de88d67a3fbb9ff47bf7dc42bb977ec539325R22

Reply:
Make that change and you should be good to release.